examples/redMethod/main.libsonnet

```diff
@@ -1,4 +1,4 @@
-local red = import 'redDashboard/main.libsonnet';
+local red = import './lib/redDashboard/main.libsonnet';
```

Comment:
Just want to get the smoke tests in, we can refactor the unit test/smoke test setup later.